### DISCUSSION TITLE
Is it possible to synchronize more than one directory?

### DISCUSSION THREAD

**Discussion by danilodorgam** [abraunegg/onedrive#2657] [General]
Currently, I have this configuration and would like to know if I can pass more than one folder as a parameter to 'sync_dir'. For example, 'sync_dir = "/folder1|/folder2"' or something similar.

**Comment by abraunegg** [comment]
@danilodorgam 
You cannot pass multiple directories to 'sync_dir'

You need to either store all the data in your configured 'sync_dir' or use symbolic links to reference the data there.
